app/templates/eventView.html

```diff
@@ -30,13 +30,13 @@ <h1 id="title">{{page_title}}</h1>
          </div>
     {% endif %}
     {% if isPastEvent %}
-        <p><font size="+2"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and was located in <b>{{eventData.location}}</b>.</font></p>
+        <p class="heading-size"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and was located in <b>{{eventData.location}}</b>.</p>
     {% else %}
-        <p><font size="+2"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</font></p>
+        <p class="heading-size"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</p>
     {% endif %}
             <div>
                 <div>
-                    <font size="+2"><h3><br>Description:</h3>{{eventData.description}}<br><br>
+                    <h3><br class="heading-size">Description:</h3>{{eventData.description}}<br><br>
```

Comment:
having the class on the break tag is kind of weird. There's no text in the break.